README.md

```diff
@@ -32,11 +23,31 @@ How-To Tutorial videos available on the [SAS video portal](https://video.sas.com
 * [Scoring, publishing and comparing models](http://players.brightcove.net/1872491364001/default_default/index.html?videoId=6160664319001)
 * [Performance reporting](http://players.brightcove.net/1872491364001/default_default/index.html?videoId=6160664560001)
 
-## Samples
-The [samples](samples/) directory contains sample data and models for Python and R programming language models.  
+For how-to tutorial videos about using the PZMM Python package or module to register a Python model, see [PZMM Module Demos](https://github.com/sassoftware/python-sasctl/blob/master/src/sasctl/pzmm/README.md#demos) in the sasctl GitHub Repository. 
 
 ## Examples
-The [examples](examples/) directory contains examples of Jupyter notebooks and Python code that can be used to perform SAS Open Model Manager tasks.
+
+* Jupyter notebooks and Python code examples that you can use to perform model management tasks are available in the [examples](examples/) directory.
+* Sample data and models for Python and R programming language models are available in the [samples](samples/) directory.
+
+For additional examples that demonstrate how to use the sasctl PZMM module to import Python models, see the [examples](https://github.com/sassoftware/python-sasctl/tree/master/examples) directory in the sasctl GitHub repository.
```

Comment:
If we are pointing at pzmm as a module of sasctl, let's change the language to match the import statement:

~~sasctl PZMM~~ sasctl.pzmm

Reply:
As above, if referencing the full repository, sasctl should be **python-sasctl**.

Reply:
Normally we have a generic form/descriptive text to use, and not the actual code statement.
"pzmm module of the sasctl package"  or if you still prefer, I can say "how to use the sasctl.pzmm Python module"

Reply:
I think 'pzmm module of the sasctl package' works.